.github/workflows/ci.yml

```diff
@@ -145,3 +147,10 @@ jobs:
           # message to the pull request title
           # https://github.com/cypress-io/github-action/issues/124#issuecomment-653180260
           COMMIT_INFO_MESSAGE: ${{ github.event.pull_request.title }}
+
+          # Percy config
+          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
+          PERCY_PARALLEL_TOTAL: 3
+          PERCY_PARALLEL_NONCE: '${{ github.event_name }}-${{ github.sha }}'
```

Comment:
The Percy docs say that `PERCY_PARALLEL_NONCE` does not need to be set in most cases.
Are we sure we need this property, maybe Percy can figure it out itself?

Quote from the docs: [^docs]

> `PERCY_PARALLEL_NONCE`: A unique identifier used to group parallel builds. This can be any string that is shared between parallel runners, such as the CI build number.
>
> -In most setups this is automatically discovered and does not need to be set. See Parallel test suites for more info.

[^docs]: https://docs.percy.io/docs/environment-variables#optional

Reply:
I didn't realize about this. Let me remove it to see if it works as expected.

Reply:
Everything keeps working as expected! Good catch.